project/civiwiki/settings.py

```diff
@@ -22,6 +25,11 @@ def get_env_variable(environment_variable, optional=False):
             raise ImproperlyConfigured(error)
 
 
+sentry_sdk.init(
+    dsn="https://4107ae45a36e495e8c4a85a4cbcdea00@sentry.io/1413260",
```

Comment:
Should this be stored in an environment variable, rather than hard-coded in our sourcecode? Also, other CiviWiki deployments, if any, may not use this feature. So, it should be enabled only if the environment variable is present.

Reply:
Oh, you're right! That's a great catch, thank you!